apps/laboratory/src/pages/library/verify-valid.tsx

```diff
@@ -0,0 +1,56 @@
+import { createWeb3Modal } from '@web3modal/wagmi/react'
```

Comment:
How is this different from our regular pages?

Reply:
The metadata URL is set to `http://localhost:3000` and it uses a special project ID with Verify API enabled on localhost